### ISSUE TITLE
Error: connect ENOENT /bigquery/v2/projects/<project>/jobs

### ISSUE THREAD

**Issue by SJAnderson** [googleapis/nodejs-bigquery#243]
#### Environment details

  - OS: OSX Mojave 10.14
  - Node.js version: 10.10.0 (using experimental-modules)
  - npm version: 6.4.1 
  - yarn version: 1.10.1 (this crashes using either yarn or npm)
  - `@google-cloud/bigquery` version: ^2.0.0

#### Steps to reproduce

We were previously using `@google-cloud/bigquery@1.0.0` and updated to v2, and the `bigQuery` instance crashes on `createQueryJob`. **This functioned fine on version 1.**

1. Create a service-account with the `BigQuery Admin` role. Download the JSON key file.
2. Create a `package.json` with the following, install using `npm` or `yarn`.

```json
{
  "dependencies": {
    "@google-cloud/bigquery": "^2.0.0"
  },
  "scripts": {
    "start": "node --experimental-modules index.mjs"
  }
}

```

3. create an index.mjs file below, updating `<project>` and `<path_to_keyfile>.json` with your appropriate values

```es6
// index.mjs
import BQ from '@google-cloud/bigquery';
const {BigQuery} = BQ;

const bigQuery = new BigQuery({
  keyFilename: '<path_to_keyfile>.json',
  projectId: '<project>',
});

const run = async () => {
  // crashes below
  const job = await bigQuery.createQueryJob({
    query: '<query_string>',
    useQueryCache: true,
  })
  // there's more below this but we never make it past the above call
}

run()
  .then(() => {
    process.exit();
  })
  .catch((err) => {
    console.warn(err);
    process.exit(1);
  })
```

4. execute via `yarn start` or `npm start`. See that it matches the error below.

### Error printed

```js
{ Error: 'connect ENOENT /bigquery/v2/projects/<project>/jobs at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)',
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'connect',
  address: '/bigquery/v2/projects/<project>/jobs' }
```

**Comment by JustinBeckwith** [comment]
Greetings!  This is a weird question - but can you try running `npm install request`, and see if that fixes the issue?

**Comment by SJAnderson** [comment]
@JustinBeckwith tested with yarn and npm. That does fix the issue but shouldn't that be a dependency in this module if its required to work with this version?

For others reading this:
- Successful execution of `yarn add request` and `npm install request --save`  fixed this issue
- Confirmed `request` was required because uninstalling it would reproduce the issue of this ticket

**Comment by JustinBeckwith** [comment]
Awesome.  Well then... good news!  This got fixed in #244 :)  We cut 2.0.1 which should resolve the issue yesterday. Thank you for helping us confirm the fix!
